ArmState arm
float64 armVelocity
